### PR TITLE
Class based views addition to advanced testing topics

### PR DESCRIPTION
**Pull request by bee-keeper**
Small addendum to show how to test CBV with request from request
factory

### CODE REVIEW

File: docs/topics/testing/advanced.txt [hunk 1]
```diff
@@ -63,6 +63,8 @@ The following is a simple unit test using the request factory::
 
             # Test my_view() as if it were deployed at /customer/details
             response = my_view(request)
+            # Use this syntax for class based views
```
Comment (timgraham) [inline docs/topics/testing/advanced.txt]: "class-based"

File: docs/topics/testing/advanced.txt [hunk 2]
```diff
@@ -63,6 +63,8 @@ The following is a simple unit test using the request factory::
...
+            response = my_view.as_view()(request)
```
Comment (timgraham) [inline docs/topics/testing/advanced.txt]: a class would have a name like "MyView"

**Comment by bee-keeper** [comment]
^

**Comment by timgraham** [comment]
merged in a17216646adaaa909fe52a4e974b56b4f094fc72, thanks.

**Pull request closed without merging**